.circleci/integration-test.py

```diff
@@ -158,8 +158,8 @@ def main():
     run_test_parser = subparsers.add_parser('run-test')
     run_test_parser.add_argument('--installer-args', default='')
     run_test_parser.add_argument('--upgrade', action='store_true')
+    run_test_parser.add_argument('--bootstrap_pip_spec', nargs='?', default="", type=str)
```

Comment:
Wondering whether the parameter should be named `--bootstrap-pip-spec`, similar to `--installer-args`? (so they both follow the same style)

Reply:
Oh, right :confused: Yes, that would be better.